structtest/struct.c

```diff
@@ -0,0 +1,306 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
If this file is only for testing, rename this to `struct_test.c`

Reply:
It is only for testing. The other C files that are used for testing don't have `_test.c` so those should be changed in a different PR.

Reply:
Sure, thanks